js/core/utils/size.js

```diff
@@ -135,6 +135,8 @@ export const parseHeight = function(value, container) {
         value = parseInt(value);
     } else if(value.indexOf('vh') > 0) {
         value = window.innerHeight / 100 * parseInt(value.replace('vh', ''));
+    } else if(element && value.indexOf('em') > 0) {
+        value = parseFloat(value.replace('em', '')) * parseFloat(window.getComputedStyle(element).fontSize);
```

Comment:
Are u sure it's ok for performance?

Reply:
Getting this object doesn't require that a full recalc is performed, only its getters will force it. To get the `font-size` property is more light operation if we will compare with calling of the `width` getter.